contracts/templates/FairSaleTemplate.sol

```diff
@@ -102,7 +102,8 @@ contract FairSaleTemplate is AquaTemplate {
             tokenOut,
             tokensForSale,
             tokenSupplier,
-            encodedInitData
+            encodedInitData,
+            address(this)
```

Comment:
We can simplify it, by emitting msg.sender on the saleLauncher event instead of sending the address along

Reply:
I removed it, but forgot to push 😅 8ff9726

Reply:
@adamazad awesome! :)